deployment/clusterCmd.py

```diff
@@ -38,6 +38,14 @@
 
 class ClusterCmd():
     def register(self, parser):
+
+        logger.warning("Kubernetes deployment in paictl maybe be deprecated in the future!")
```

Comment:
"maybe be deprecated" --> "will be deprecated"

Reply:
Done